### ISSUE TITLE
Feature request: Keyboard navigation (iPad OS)

### ISSUE THREAD

**Issue by tkornack** [zulip/zulip-flutter#1605]
It is quite common to use an iPad with a keyboard. In this configuration, I was unable to perform even rather basic keyboard navigation and most notably none of the usual cmd-return, shift-return or tab return could send a message. Would be lovely to see it implemented here.

**Comment by chrisbobbe** [comment]
We've had another request for this today:

[#mobile > New app: can’t send message on iPad Apple magic keyboard @ 💬](https://chat.zulip.org/#narrow/channel/48-mobile/topic/New.20app.3A.20can.E2.80.99t.20send.20message.20on.20iPad.20Apple.20magic.20keyboard/near/2205836)

> I’m on iPad with the Apple magic keyboard. There doesn’t seem to be a way to send a message other than clicking on the “send” icon. Return / Shift+Return / or any other combination I’ve tried fails to send the message.

**Comment by shubham-padia** [comment]
When working on this issue, it would be good for the author to check how compose box on web does undo for reverse linkification, for channel/topic links and some other links and try to replicate it in this issue or a follow-up issue: [#mobile-design > Undo paste for reverse linkification @ 💬](https://chat.zulip.org/#narrow/channel/530-mobile-design/topic/Undo.20paste.20for.20reverse.20linkification/near/2500337)
